### PR TITLE
MAC removed from logs uploaded to Dasharo cloud

### PR DESCRIPTION
**Pull request by kotylamichal**
- Resolves Dasharo/dasharo-issues#92

### CODE REVIEW

File: meta-dts-distro/recipes-dts/reports/dasharo-hcl-report/dasharo-hcl-report [hunk 1]
```diff
@@ -135,8 +135,19 @@ filename+="_$(date +'%Y_%m_%d_%H_%M_%S_%N')"
 filename="${filename// /_}"
 
 echo "Creating archive with logs..."
-tar -zcf "$filename.tar.gz" logs/*
-rm -rf logs
+if [ "$SEND_LOGS" = "true" ]; then
```
Comment (TomaszAIR) [inline meta-dts-distro/recipes-dts/reports/dasharo-hcl-report/dasharo-hcl-report:138]: @kotylamichal lets just remove the mac address from the logs without saving it in local copy.

**Comment by TomaszAIR** [comment]
@kotylamichal are there any other sensitive informations worth dropping?

**Comment by kotylamichal** [comment]
> @kotylamichal are there any other sensitive informations worth dropping?

In my opinion, any other data cannot be recognized as sensitive or confidential. For sure MAC address doesn't exist in other log files which we collect.

File: meta-dts-distro/recipes-dts/reports/dasharo-hcl-report/dasharo-hcl-report (hunk 1 above)
Comment (kotylamichal) [reply meta-dts-distro/recipes-dts/reports/dasharo-hcl-report/dasharo-hcl-report]: @TomaszAIR done

**Comment by TomaszAIR** [comment]
Merging it to hcl-report-improvements. Edited issue description. adding `- Resolves Dasharo/dasharo-issue#NUMBER` we are able to automate closing given issues while merging the PR

**Comment by TomaszAIR** [comment]
Merging to hcl-report-improvements. Added `resolves` to description. Thanks to this this PR will be mentioned in issue. When we merge it to the main, issue should be automatically closed.

**Pull request merged**